libs/internal/src/events/asio_event_processor.cpp

```diff
@@ -255,7 +255,7 @@ std::vector<OutputEvent> AsioEventProcessor<SDK>::Process(
 
                 if (event.require_full_event) {
                     out.emplace_back(FeatureEvent{std::move(base),
-                                                  event.context.KindsToKeys()});
```

Comment:
To fix the formatting errors, you can run clang format (maybe part of clang package on Arch?)